Automattic/woocommerce-payments
# Display red badge after 3 days

Fixes #2837 

#### Changes proposed in this Pull Request

Updates the red badge to display after 3 days instead of 7.


![Screen Shot 2021-10-13 at 5 02 38 PM](https://user-images.githubusercontent.com/10561050/137212554-fa68bc0c-c6d1-46e5-928a-2465495797fe.png)
#### Testing instructions

1. Spin up a new site (or delete `wcpay_menu_badge_hidden`)
2. Do not connect stripe
3. Wait 3 days (or modify `wcpay_activation_timestamp` to be a timestamp >3 days ago)
4. Check that the red badge is shown next to "Payments" in the menu



-------------------


**Post merge**
